FCCee/ALLEGRO/compact/ALLEGRO_o1_v03/DectDimensions.xml

```diff
@@ -18,36 +18,39 @@
     <constant name="world_side" value="6100*mm"/>
     <constant name="CrossingAngle" value="0.030*rad"/>  
 
-    <constant name="GlobalTrackerReadoutID" type="string" value="system:5,layer:3,stave:6,module:5,sensor:2"/>
-    <constant name="GlobalTrackerReadoutID_OB" type="string" value="system:5,layer:1,stave:6,module:4,sensor:2"/>
-    <constant name="GlobalTrackerReadoutID_Disk" type="string" value="system:5,side:-2,layer:3,petal:3,stave:6,module:10,sensor:2"/>
-    <constant name="GlobalTrackerReadoutID_SiWrapperB" type="string" value="system:5,layer:1,stave:9,module:6,sensor:1"/>
-    <constant name="GlobalTrackerReadoutID_SiWrapperD" type="string" value="system:5,side:1,layer:1,module:6,sensor:1"/>
+    <constant name="GlobalTrackerReadoutID_DCH" type="string" value="system:1,layer:16,phi:16"/>
```

Comment:
The whole point of the "Global" tracker readout ID is having a global one for all tracker readouts.

Reply:
My mistake for adding it here, apparently it is defined directly in the DriftChamber XML:
https://github.com/key4hep/k4geo/blob/bfe18998e04bafb3ccdf443f236c2b1b460f44c7/FCCee/IDEA/compact/IDEA_o1_v03/DriftChamber_o1_v02.xml#L127